src/recon.rs

```diff
@@ -2752,7 +2785,10 @@ pub(crate) unsafe fn rav1d_recon_b_intra<BD: BitDepth>(
                             if debug_block_info!(f, t.b) {
                                 println!(
                                     "Post-y-cf-blk[tx={},txtp={},eob={}]: r={}",
-                                    intra.tx as c_int, txtp as c_uint, eob, ts.msac.rng,
+                                    intra.tx as c_int,
+                                    txtp as c_uint,
+                                    eob,
+                                    ts_c.as_deref().unwrap().msac.rng,
```

Comment:
For all of these `.as_deref()`s and `.as_deref_mut()`s, I think they can just be `.as_ref()` and `.as_mut()`s (maybe).  `.as_deref()` is like `.as_ref()` but is also doing a deref, so it's helpful from going from `Option<String>` to `Option<&str>` instead of `Option<&String>`, but I don't think we need that extra deref here.

Reply:
Ahh, okay then your way should be fine.  Didn't realize we'd get the double reference with my way.